Bermuda/Bermuda/StatusTracker.cpp

```diff
@@ -15,7 +14,13 @@ void StatusTracker::init()
 	achievements.push_back(new Achievement("Ice mined"));
 	achievements.push_back(new Achievement("Cacti chopped"));
 	achievements.push_back(new Achievement("Rocks mined"));
-	achievements.push_back(new Achievement("Trees cut"));
+	achievements.push_back(new Achievement("Trees logged"));
```

Comment:
"Trees logged" is perfect correct engels. Maar misschien kun je er beter "Trees chopped" van maken aangezien we ook bij cactussen "chopped" gebruiken. :)